python/sklearn-digits/app.py

```diff
@@ -107,22 +98,29 @@ def evaluate(self, file_path: str) -> Tuple[EvaluateResult, List[EvaluateResultD
             res = EvaluateResult(num, accuracy, p_r_f[0].tolist(), p_r_f[1].tolist(), p_r_f[2].tolist(), self.labels)
             return res, details
         except Exception as e:
-            self.logger.error(str(e))
-            self.logger.error(traceback.format_exc())
+            self.system_logger.error(str(e))
+            self.system_logger.error(traceback.format_exc())
             return EvaluateResult(), []
 
-    def get_evaluate_detail(self, file_path: str, results: List[EvaluateResultDetail]) -> Generator[EvaluateDetail, None, None]:
+    def get_evaluate_detail(self, filepath: str, details: List[EvaluateResultDetail]) -> Generator[EvaluateDetail, None, None]:
         """ override
-        Create EvaluateDetail by merging evaluation data from file_path and EvaluateResultDetail
-
-        :param file_path: Evaluation data file path. str
-        :param results: Detail result of each prediction
-        :return:
-            detail: Evaluation data & result of each prediction
+        get_evaluate_detail
+        :param filepath: Evaluation data file path. str
+        :param details: Detail result of each prediction. List[EvaluateResultDetail]
+        :return rtn: Return results. Generator[EvaluateDetail, None, None]
+            rtn.input: Input data. PredictInput, one of string/bytes/arr[int]/arr[float]/arr[string]
+            rtn.label: Predict label. PredictLabel, one of string/bytes/arr[int]/arr[float]/arr[string]
+            rtn.result: Predict detail. EvaluateResultDetail
         """
         try:
-            for i, (correct_label, data) in enumerate(self.__generate_eval_data(file_path)):
-                yield EvaluateDetail(input=data, label=correct_label, result=results[i])
+            for i, (correct_label, data) in enumerate(self.__generate_eval_data(filepath)):
+                yield EvaluateDetail(input=data, label=[correct_label], result=details[i])
```

Comment:
we should match label's types. (label's type in here is array of string and label's type in `predict` method is string)

Reply:
I have updated!